src/fhir/1_Questionnaire-C-IDAS.json

```
@@ -181,7 +181,7 @@
       "type": "choice",
       "required": false,
       "linkId": "cs-idas-3",
-      "text": "Lack of reactivity to pleasant events",
+      "text": "In the last week have you observed a Lack of reactivity to pleasant events in your care recipient?",
```

not sure if "Lack" should be capitalized here.
```suggestion
      "text": "In the last week have you observed a lack of reactivity to pleasant events in your care recipient?",
```